submissions/asaMitaka/dom/style.css

```diff
@@ -0,0 +1,126 @@
+/* content */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    width: 100%;
+    height: 100%;
+}
+
+header {
+    display: flex;
+    justify-content: center;
+    align-self: center;
+    font-size: 30px;
+}
+
+.content {
+    display: flex;
+    justify-content: space-evenly;
+    height: 96vh; 
+}
+
+/* asideItem */
+
+.asideBlock {
+    width: 300px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+}
+
+.asideBtn {
+    width: 200px;
+    height: 50px;
+    border: 1px solid black;
+    background-color: gainsboro;
+    cursor: pointer;
+}
+
+.asideBtn:hover {
+    background-color: azure;
+    border: 1px solid blue;
+}
+
+/* mainItems */
+
+.mainBlock {
```

Comment:
Make it much wider so that the content does not get out of the block, also it fixes problem with the height of some albums)
![image](https://user-images.githubusercontent.com/16196199/183964040-ed68c424-d067-48f2-bc49-52eba4b61355.png)

Reply:
Add some padding to your list)
![image](https://user-images.githubusercontent.com/16196199/184121509-625fdd33-251b-418b-aaae-0cfdf83bbb52.png)